src/otx/algo/instance_segmentation/heads/maskdino_encoder.py

```diff
@@ -347,25 +277,26 @@ def __init__(
 
         use_bias = norm == ""
         for idx, in_channels in enumerate(self.feature_channels[: self.num_fpn_levels]):
-            lateral_norm = get_norm(norm, conv_dim)
-            output_norm = get_norm(norm, conv_dim)
+            lateral_norm = nn.GroupNorm(32, conv_dim)
+            output_norm = nn.GroupNorm(32, conv_dim)
 
-            lateral_conv = Conv2d(
+            lateral_conv = Conv2dWithNormActivation(
                 in_channels,
                 conv_dim,
                 kernel_size=1,
                 bias=use_bias,
                 norm=lateral_norm,
+                activation=activation,
             )
-            output_conv = Conv2d(
+            output_conv = Conv2dWithNormActivation(
```

Comment:
Why has this module been changed? I vote for using our current Conv functionality. We have already refactored it previously to use globally for all our tasks and models. It supports norm layers overriding and activations as well.